academy/ibc/what-is-ibc.md

```diff
@@ -0,0 +1,84 @@
+
+</HighlighBox>
+
+## Internet of blockchains
+
+IBC is essential for application-specific blockchains like Cosmos. It offers a standard communication channel for applications on two different chains that need to communicate with each other.
+
+More details on the specifications will follow in the next section, but notice that IBC is not limited to Cosmos blockchains. Solutions can even be found for cases where some requirements are not initially met. An example of such cases is the use of IBC for Proof-of-Work (PoW) blockchains like Ethereum, as a PoW consensus algorithm which does not ensure finality. Ethereum-compatibility is enabled by creating a peg-zone where probabilistic finality is considered deterministic (i.e. irreversible) after a given threshold of block confirmations.
+
+![Internet of blockchains](images/internetofchains.png)
+
+Although application-specific blockchains offer superior scalability compared to general-purpose blockchain platforms, smart contract development for general-purpose chains and generic virtual machines (VMs), e.g. Ethereum, offer their own benefits. IBC provides a method of incorporating the strengths of general-purpose and application-specific blockchains into unified overall designs. It allows, for example, a Cosmos chain tailored towards performance and scaleability to use funds that originate on Ethereum and possibly record events in a Corda distributed ledger or, in the reverse, a Corda ledger initiating the transfer of underlying assets defined in Cosmos or Ethereum.
+
+New concepts such as Interchain Accounts will continue to further increase adoption times and the possibilities IBC offers: a blockchain will be able to control an account on another blockchain with Interchain Accounts.
+
+## Architecture
+
+Assume we have two chains: **chain *A* and chain *B***. Both will implement the ICS for IBC.
+
+![IBC overview](images/ibcoverview.png)
+
+Notice three crucial elements in the diagram above:
+
+* The chains use relayers to communicate;
+* Chains can use different relayers to send messages to each other;
+* Each side of the relay uses the light client of the other chain to quickly verify incoming messages.
+
+In fact, the relayer listens and picks up messages from a channel to forward them. Both chains implement IBC and first initiate a connection to then establish channels. Notice that relayers ensure that both chains know that a package has been transmitted.
```

Comment:
Imo the packet flow probably fits in a further section that goes into more depth for the Cosmos SDK dev (deep dive) whereas this I believe is meant as an intro for the technical decision maker (fast track)... if I'm not mistaken. 

But maybe this last sentence should be rewritten or postponed as up until this point we have not really introduced the layers of abstraction...
From this diagram the main takeaways for the CTO (feel a bit weird about that as the CTO should also do the deep dive sections imo) would be: 

- Both chains need a way to reason about the state of the other => light clients (see more below)
- Both chains cannot directly communicate with each other but packet relay is provided by relayer operators as an important part of the infrastructure
- Relaying is permissionless & multiple relayer operators can serve the same pair of chains (if we want to stay clear of the terms connections and channels for now)
- Quick mention of security while referring to deep dive section for more details?? As for a technical decision maker this will likely be more important than knowing exact packet flow I believe

A good directive would be to only show what is explicitly shown on the diagram